### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by SmartManoj**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -86,9 +87,9 @@ DATADIR = "pocketsphinx/test/data"
 
 # Create a decoder with certain model
 config = Decoder.default_config()
-config.set_string('-hmm', path.join(MODELDIR, 'en-us/en-us'))
```
Comment (nshmyrev) [inline readme.md:89]: This change doesn't seem correct. Our models are in pocketsphinx/model/en-us/en-us for example, not in pocketsphinx/model/en-us.

**Review by nshmyrev — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (SmartManoj) [reply readme.md:89]: My pocketsphinx  version is 0.1.3 on Windows-10 32bit.
There is only one en-us folder

Comment (SmartManoj) [reply readme.md:89]: Source:https://pypi.python.org/pypi/pocketsphinx

Comment (nshmyrev) [reply readme.md:89]: This is not our software

**Pull request closed without merging**